public/Electron/utils.js

```diff
@@ -14,7 +14,7 @@ const checkPortNumber = () => {
   ).toString();
   const lines = stdout.split(os.EOL);
 
-  if (lines.at(-1) === "") {
+  if (lines[lines.length - 1] === "") {
```

Comment:
```suggestion
  if (lines.at(-1) === "") {
```
`.at(-1)`을 쓰면 배열의 마지막요소라는 것이 더 잘 보이지 않을까 생각이 들었습니다.